aplus_setup.py

```diff
@@ -49,6 +49,7 @@ def setup(app):
     app.add_config_value('enable_ref_link_multilang_suffix_correction', False, 'html')
     app.add_config_value('reveal_submission_feedback', None, 'html')
     app.add_config_value('reveal_model_solutions', None, 'html')
+    app.add_config_value('autosave', False, 'html')
```

Comment:
In the README, this was named `enable_autosave`. I think it makes sense to use the names `enable_autosave` in conf.py and the `autosave` flag in the questionnaire directive.